cpp/ps0002/p0002-soln.cpp

```diff
@@ -1,29 +1,40 @@
-// عدد سكان عمارة
-int populationBuilding= 100;
-
-// عدد البدائل لدوا في الصيدلية
-int drug_Alternatives = 5;
-
-// أفراد تيم في شركة برمجة
-int team_Members = 8;
-
-// سعر واحدة من مخبوزات الفرن
-float bakery_Price = 3.44;
-
-// الحرف الأول من اسم شخص
-char first_char= 'A';
-
-// عدد المواليد في دولة تعدادها كبير زي الصين
-long long chinaBirths = 4985700000;
-
-// ناتج اكثر دقة لقسمة رقم على 7
-//تنفعfloat or double
-//بس double اكثر دقة
-double division_Result = 10.0 / 7.0;
-
-// فاتورة الكهرباء لخط انتاج في مصر
-//تنفعfloat or double
-double electricity_Bill = 1500.75;
-
-// عمر أشخاص كبار و صغار في السن بالثانية
-long long age_Seconds = 31536000; 
+#include <iostream>
+using namespace std;
+
+int main()
+{
+  // عدد سكان عمارة
+  short populationBuilding = 100; // 0.5/0.75
+  
+  // عدد البدائل لدوا في الصيدلية
+  short drug_Alternatives = 5; // 0.75/0.75
+  
+  // أفراد تيم في شركة برمجة
+  short team_Members = 8; // 0.75/0.75
+  
+  // سعر واحدة من مخبوزات الفرن
+  float bakery_Price = 3.44; // 0.75/0.75
+  
+  // الحرف الأول من اسم شخص
+  char first_char= 'A'; // 0.75/0.75
+  
+  // عدد المواليد في دولة تعدادها كبير زي الصين
+  int chinaBirths = 4985700; // 0.25/0.75
+  
+  // ناتج اكثر دقة لقسمة رقم على 7
+  //تنفعfloat or double
+  //بس double اكثر دقة
+  double division_Result = 10.0 / 7.0; // 0.75/0.75
+  
+  // فاتورة الكهرباء لخط انتاج في مصر
+  //تنفعfloat or double
+  double electricity_Bill = 1500.75; // 0.75/0.75
+  
+  // عمر أشخاص كبار و صغار في السن بالثانية
+  unsigned int age_Seconds = 31536000; // 0.5/0.75
+
+  // total 5.75/6.75

```

Comment:
- [x] all variables required in the document are declared
- [ ] variables are declared with the correct data type
- [x] using correct literals and notations for assigning initial values to the variables